Update device-compliance-get-started.md (#3075)
MicrosoftDocs/memdocs: memdocs/intune/protect/device-compliance-get-started.md

| **Maximum OS version** | - **Android 4.0 and later**: Quarantined<br>- **Samsung Knox Standard 4.0 and later**: Quarantined<br>- **Android Enterprise**: Quarantined<br><br>- **iOS 8.0 and later**: Quarantined<br>- **macOS 10.11 and later**: Quarantined<br><br>- **Windows 8.1 and later**: Quarantined |
| **Windows health attestation** | - **Android 4.0 and later**: Not applicable<br>- **Samsung Knox Standard 4.0 and later**: Not applicable<br>- **Android Enterprise**: Not applicable<br><br>- **iOS 8.0 and later**: Not applicable<br>- **macOS 10.11 and later**: Not applicable<br><br>- **Windows 10/11**: Quarantined<br>- **Windows 8.1 and later**: Quarantined |

> [!NOTE]

> > The Company Portal enters the enrollment remediation flow when the user logs into CP and the device has not successfully checked in with Intune in 30+ days (or the device is noncompliant due to a "Lost contact" compliance reason). In this flow, we attempt to initiate a checkin one more time. If that still does not succeed, then we issue a retire command to allow the user to re-enroll the device manually.

Reviewer: ```suggestion
> The Company Portal app enters the enrollment remediation flow when the user signs into the Company Portal app and the device has not successfully checked in with Intune for 30 days or more (or the device is noncompliant due to a _Lost contact_ compliance reason). In this flow, we attempt to initiate a check-in one more time. If that still does not succeed, we issue a retire command to allow the user to re-enroll the device manually.
```
